src/components/RestCourses/AddCourseForm.js

```diff
@@ -124,7 +124,14 @@ function AddCourseForm(props) {
         <Modal
           visible={props.visible}
           onCancel={props.hideModal}
-          onOk={submitForm}
+          onOk={e => {
+            if (disabled) {
+              props.hideModal();
```

Comment:
do we really want to close the modal `onOk` if the values our user inputted are disabled? I would assume maybe we show them the error messages to clarify that their input is invalid

Reply:
I mean, the error messages are there on the form if the data is not correct. This was just built to cover our backend just in case a user hits ok instead of cancel and it essentially does the same thing as cancel, if the data is not successful. I guess we could add an alert, but this was just a fix for now.